README.md

```diff
@@ -287,6 +287,46 @@ onmessage = (event) => {
 };
 ```
 
+## Node.js
+
+You can run highlight.js on node to prepare content before sending it to the browser.
+
+
+```js
+// require the highlight.js library including all languages
+const hljs = require('./highlight');
```

Comment:
I know it's an example but should this be `highlight.js` just for consistency with the below example?